### PR TITLE
Remove selenium tests

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Disable old selenium tests that are always failing and will be removed anyway.
| Type?         | bug fix
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Tests should be green.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15590)

### CODE REVIEW

**Comment by matks** [comment]
You the MVP @PierreRambaud

**Review by matks — APPROVED** [review]

File: tests/E2E/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "parsed-url": "0.0.0",
     "q": "^1.4",
     "req": "^0.1",
-    "selenium-standalone": "^6.14.0",
+    "selenium-standalone": "^6.16",
```
Comment (mickaelandrieu) [inline tests/E2E/package.json:38]: why do we need selenium if we remove the selenium test suite ?

File: tests/E2E/test/globals.webdriverio.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ global.URL = siteUrl;
 global.install_language = argv.LANGUAGE || 'en';
 global.install_country = argv.COUNTRY || 'france';
 global.country_currency = argv.CURRENCY || 'EUR';
-global.db_server = argv.DB_SERVER || 'lolcahost';
+global.db_server = argv.DB_SERVER || 'localhost';
```
Comment (mickaelandrieu) [inline tests/E2E/test/globals.webdriverio.js:19]: :+1:

File: tests/E2E/package.json (hunk 1 above)
Comment (PierreRambaud) [reply tests/E2E/package.json:38]: Selenium is still needed to run our tests under `tests/E2E`. We only remove `tests/Selenium`

**Pull request merged**